docs/source/en/_toctree.yml

```diff
@@ -18,7 +18,7 @@
   - local: tutorials/basic_training
     title: Train a diffusion model
   - local: tutorials/using_peft_for_inference
-    title: Inference with PEFT
+    title: Load LoRAs for inference
```

Comment:
It makes a lot of sense now because we have made `peft` a required dependency to do all things LoRA in `diffusers` :)